src/components/RightPane/CoursePane/SearchForm/AdvancedSearch.tsx

```diff
@@ -28,7 +30,29 @@ const styles = {
     },
 };
 
-class AdvancedSearchTextFields extends PureComponent {
+interface AdvancedSearchTextFieldsProps {
+    classes?: ClassNameMap
+}
+
+interface AdvancedSearchTextFieldsState {
+    instructor: string;
+    units: string;
+    endTime: string;
+    startTime: string;
+    coursesFull: string;
+    building: string;
+    room: string;
+}
+
+interface AdvancedSearchProps {
+    classes: ClassNameMap
+}
+
+interface AdvancedSearchState {
+    expandAdvanced: boolean;
+}
+
+class ASTF extends PureComponent<AdvancedSearchTextFieldsProps, AdvancedSearchTextFieldsState> {
```

Comment:
maybe call this UnstyledAdvancedSearchTextFields or something else verbose but more descriptive. I kinda dislike acronyms. You could also just add a comment explaining it.